UI/properties-view.cpp

```diff
@@ -288,6 +288,57 @@ QWidget *OBSPropertiesView::AddText(obs_property_t *prop, QFormLayout *layout,
 		connect(edit, SIGNAL(textEdited(const QString &)), info,
 			SLOT(ControlChanged()));
 		return nullptr;
+	} else if (type == OBS_TEXT_INFO) {
+		QString desc = QT_UTF8(obs_property_description(prop));
+		const char *long_desc = obs_property_long_description(prop);
+		obs_text_info_type info_type =
+			obs_property_text_info_type(prop);
+
+		QLabel *info_label = new QLabel(QT_UTF8(val));
+
+		if (info_label->text().isEmpty() && long_desc == NULL) {
+			label = nullptr;
+			info_label->setText(desc);
+		} else
+			label = new QLabel(desc);
+
+		if (long_desc != NULL) {
+			if (info_label->text().isEmpty())
+				info_label->setText(QT_UTF8(long_desc));
+			else {
```

Comment:
Rather than two levels of conditionals, I'd prefer something like

```cpp
if (long_desc != NULL && !info_label->text().isEmpty()) {
	bool lightTheme = ...
} else if (long_desc != NULL) {
	info_label->setText(QT_UTF8(long_desc));
}
```

This should also make it easier to make the `lStr` variable easier to read.

Reply:
I made the change.